schedule/ha/bv/migration/migration_offline_sles_ha.yaml

```diff
@@ -0,0 +1,72 @@
+---
+name: migration_offline_sles_ha
+description: >
+  Offline migration from installed SLES with HA extension
+
+  This test does an offline migration using the *DVD* media, with or without SCC registration.
+vars:
+  BOOTFROM: 'd'
+  BOOT_HDD_IMAGE: '1'
+  DESKTOP: 'textmode'
+  DROP_PERSISTENT_NET_RULES: '1'
+  FULL_UPDATE: '1'
+  HDDVERSION: '%ORIGIN_SYSTEM_VERSION%'
+  INSTALLONLY: '1'
+  MAX_JOB_TIME: '14400'
+  ORIGINAL_TARGET_VERSION: '%VERSION%'
+  PATCH: '1'
+  # The -yaml suffix added below is something temporary for not overwriting qcow2 images generated by our BV tests.
+  # It will be removed when the fully YAML way will be moved to the BV job group.
+  PUBLISH_HDD_1: 'upgrade_from-%HDDVERSION%-%DISTRI%-%VERSION%-%ARCH%-Build%BUILD%-%CLUSTER_NAME%-%HOSTNAME%-HA-BV-yaml.qcow2'
```

Comment:
I am wondering if it would be better to move PUBLISH_HDD_1 to the yaml job group instead? This way you will avoid having to submit a new PR just to remove the `-yaml` and we can also use the same schedule in other groups (if necessary) or in other scenarios (if possible). WDYT?

Reply:
I agree, and by moving the PUBLISH_HDD_1 to the yaml job group, we can use the same file (migration_offline_sles_ha.yaml) for `both migration_offline_dvd` and `migration_offline_scc`

Reply:
Done